src/report.js

```diff
@@ -51,20 +52,27 @@ class Report {
     const {
       functionName,
       functionVersion,
-      awsRequestId,
       logGroupName,
       logStreamName,
       memoryLimitInMB
     } = this.context;
 
-    let { invokedFunctionArn } = this.context;
+    let { invokedFunctionArn, awsRequestId } = this.context;
 
     // Patch invokedFunctionArn in cases of SAM local invocations
     // or if invokedFunctionArn is missing
     if (invokedFunctionArn === undefined || process.env.AWS_SAM_LOCAL) {
       invokedFunctionArn = `arn:aws:lambda:local:0:function:${functionName}`;
     }
 
+    if (
+      process.env.AWS_SAM_LOCAL ||
+      awsRequestId === undefined ||
+      awsRequestId === 'id'
+    ) {
+      awsRequestId = `local-${functionName}|${uuid()}`;
```

Comment:
Like the `local` part. Including the function name is probably redundant as it's both in the `functionName` and `invokeFunctionArn`.

Reply:
Oh, cool--thanks! I wasn't sure if having the name here would make this easier to scan visually, but this makes sense.  I'll take that out here and in the test.

Reply:
functionName removed and PR updated.